.github/workflows/static-analysis.yml

```diff
@@ -7,21 +7,19 @@ on:
     branches:
       - main
   schedule:
-    - cron: "0 0 * * *"
+    - cron: '0 0 * * *'
 
 jobs:
   phpstan:
-    name: Static analysis with PHP ${{ matrix.php-versions }} - ${{ matrix.composer-dependencies }} dependencies - ${{ matrix.os }} OS
-    runs-on: ${{ matrix.os }}
+    name: PHPStan
+    runs-on: ubuntu-latest
     strategy:
-      fail-fast: true
       matrix:
-        php-versions: ['8.0', '8.1']
-        composer-dependencies: ['highest', 'lowest']
-        os: ['ubuntu-latest']
+        php-versions: ['8.1']
+        composer-dependencies: ['highest']
```

Comment:
I think that static analysis should stay as it to prevent future BC breaks and new features, `8.2` cannot be supported but I think that we should keep both versions.